xm-commons-domain-event/src/main/java/com/icthh/xm/commons/domain/event/domain/converter/MapToStringConverter.java

```diff
@@ -0,0 +1,49 @@
+package com.icthh.xm.commons.domain.event.domain.converter;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+@Slf4j
+@Converter
+public class MapToStringConverter implements AttributeConverter<Map<String, Object>, String> {
+
+    private ObjectMapper mapper = new ObjectMapper();
+
+    public MapToStringConverter() {
+        mapper.registerModule(new JavaTimeModule());
+    }
+
+    @Override
+    public String convertToDatabaseColumn(Map<String, Object> data) {
+        try {
+            return mapper.writeValueAsString(data != null ? data : new HashMap<>());
+        } catch (JsonProcessingException e) {
+            log.warn("Error during JSON to String converting", e);
```

Comment:
should we suppress errors here? Looks like we need to re-throw exception.
@vkirichenko what do you think?